docs/en/docs/tutorial/path-params-numeric-validations.md

```diff
@@ -84,7 +84,7 @@ The same applies for:
 
 Number validations also work for `float` values.
 
-Here's where it becomes important to be able to declare <abbr title="greater than"><code>gt</code></abbr> and not just <abbr title="greater than or equal"><code>ge</code></abbr>. As with it you can require, for example, that a value must be greater than `0`, even if it is less than `1`.
+Here's where it becomes important to be able to declare <abbr title="greater than"><code>gt</code></abbr> and not just <abbr title="greater than or equal"><code>ge</code></abbr>. As with int you can require, for example, that a value must be greater than `0`, even if it is less than `1`.
```

Comment:
It should be: As with `int` values, you can require, ...

Reply:
@andrewgin It would be good to update your PR and wrap "int" with backticks : \`int\`